node/style.scss

```diff
@@ -1,163 +1,3 @@
-}
-
-$offset: 1.8 * $unit;
-
-.app {
-  @include no-webkit-scrollbar;
-  padding: ($appbar-height + 2) $offset $offset;
-  background-color: #f5f5f5;
-  > section, > div > section {
-    padding: $offset;
-    margin-bottom: $offset;
-    background-color: #fff;
-    box-shadow: 0 1px 2px rgba(0,0,0, 0.25);
-    > h5 {
-      color: rgb(48, 63, 159);
-      &:not(:first-child) {
-        margin-top: 2 * $offset;
-      }
-    }
-    > p {
-      margin: ($offset / 4) 0;
-      opacity: .6;
-    }
-    [data-react-toolbox="card"] {
-      display: inline-block;
-      margin: $offset $offset 0 0;
-    }
-  }
-}
-
-.github {
-  position: fixed;
-  top: $appbar-height - ($button-floating-height / 2);
-  right: $button-floating-height / 2;
-  z-index: $z-index-higher;
-}
- 
-.appbar {
-  display: flex;
-  > h1 {
-    flex-grow: 1;
-    font-size: 1.8 * $unit;
-    font-weight: bold;
-
-    > small {
-      font-size: 1.8 * $unit;
-      font-weight: normal;
-    }
-  }
-
-  > .title {
-    font-size: $offset;
-    font-weight: bold;
-    position: relative;
-  }
-
-  > .smallText {
-    font-size: $offset;
-    font-weight: normal;
-    @include typo-display-3($color-contrast: true);
-  }
-
-  > .button {
-    position: absolute;
-    top: $appbar-height - ($button-floating-height / 2);
-    right: $button-floating-height / 2;
-    z-index: $z-index-higher;
-  }
-}
-
-.primary {
-  background-color: $color-primary;
-}
-
-.cards {
-  margin-top: 10px;
-  background-color: #f8f8f8;
-  border: 1px solid #f4f4f4;
-}
-
-.cardsGroup {
-  position: relative;
-  display: flex;
-  flex-wrap: wrap;
-  align-items: stretch;
-  justify-content: center;
-  padding: 20px;
-  list-style: none;
-}
-
-.cardRow {
-  display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
-  .media {
-    width: 80px;
-    height: 80px;
-    margin: 16px 16px 0 0;
-  }
-  .mediaLarge {
-    width: 140px;
-    height: 140px;
-    margin: 16px;
-  }
-}
-
-.cardItem {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: space-between;
-  margin: 0 20px;
-}
-
-.cardItemContent {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
-
-.cardItemName {
-  display: block;
-  margin: 15px 0;
-  text-align: center;
-}
-
-.card {
-  width: 345px;
-}
-
-.chipTruncateWrapper {
-  width: 20rem;
-  padding: $offset / 2;
-  margin-top: $offset;
-  border: 1px solid #f4f4f4;
-}
-
-.dropdownTemplate {
-  display: flex;
-  flex-direction: row;
-}
-
-.dropdownTemplateImage {
-  display: flex;
-  width: 32px;
-  height: 32px;
-  flex-grow: 0;
-  margin-right: 8px;
-  background-color: #ccc;
-}
-
-.dropdownTemplateContent {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 2;
-}
\ No newline at end of file
+@import "~react-toolbox/lib/commons";
+// This file could be used as an Application wide style?
+// You're existing layout is confusing to me, but I hopefully you can refactor this stuff :D
```

Comment:
Yeah sorry. I just started with React.js and then stumbled into React Toolbox. I'm hacking all this stuff together trying to figure it out. I haven't remotely tried to organize any of this stuff. Some of it was lifted from different places. Thanks for your help. I'm processing the pull request now and I'll give it a try after I sort it out.